ios/FluentUI/Bottom Sheet/CommandingItem.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// An object representing a command.
+///
+/// `CommandingItem` defines the high level properties and behavior of a command. Its visual representation is determined by
+/// the `BottomCommandingController`.
+@objc(MSFCommandingItem)
+open class CommandingItem: NSObject {
```

Comment:
should commanding item be a protocol rather than nsobject so client can easily pipe through their images and titles with their backing logic?

Reply:
similar to popupmenucontroller architecture

Reply:
I wanted to do something like it, but I'm not sure how I can apply that pattern here. We need the items to notify the controller about changes, and that can't be captured by a protocol. PopupMenuController doesn't have that problem because changes to the items don't reflect in the UI after passing them to the controller.
Clients can still subclass to pipe stuff through, while keeping our property observers intact.